src/ranged.cpp

```diff
@@ -341,29 +424,52 @@ dealt_projectile_attack Creature::projectile_attack( const projectile &proj_arg,
     return attack;
 }
 
+static double erfinv( double x )
```

Comment:
The paper I lifted it from suggests it's good for single-precision accuracy. The iterative approach has epsilon=1e-6. Accuracy really isn't a big deal for our purposes here anyway, I doubt you'd see even something as huge as a 1% error.